### PR TITLE
Additional text generators

### PR DESCRIPTION
**Pull request by 1Maxnet1**
This PR adds sections about the generative models for text LLAMA 2,  GPT-4 and BLOOM to the related work section.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -89,8 +89,14 @@ \subsection{Generative models}
 
 To achieve effective transfer with minor changes to the model architecture, task-aware input is used during transformations. \cite{radford2018improving}
 
+A similar model, but in a more recent version is the GPT-4 model. The model is yet to be released by it's developer OpenAI, but is available through an API. Compared to its predecessors GPT-4 is a multimodal model, which is not only capable of using text for input but also images. The model is trained on Next-Token-Prediction and is capable and was post-training aligned to better adhere to desired behaviour. The model is capable to perform well in different professional benchmarks out of a variety of domains and can demonstrate it's performance in different languages. \cite{openai2023gpt4}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
A similar model, but in a more recent version is the GPT-4 model. The model is yet to be released by it's developer OpenAI, but is available through an API. Compared to its predecessors, GPT-4 is a multimodal model, which means, that it is not only capable of using texts for input, but also images. The model is trained on Next-Token-Prediction and was post-training aligned to better adhere to the desired behaviour. The newest GPT-model performs well in different professional benchmarks out of a variety of domains and can demonstrate it's performance in different languages. \cite{openai2023gpt4}
```

File: paper.tex [hunk 2]
```diff
@@ -89,8 +89,14 @@ \subsection{Generative models}
...
+
 The \textbf{Falcon-RW-1B} model built by TII (\url{https://www.tii.ae/}) is the second text generative model we use to test our augmentation models. It is a decoder-only model trained on 350 billion tokens of the RefinedWeb data set. This data set leverages strict filtering and stringent deduplication to uplift the quality of web data. \cite{penedo2023refinedweb}
 
+The \textbf{Llama 2} model built and released by Meta. It is available in different sizes, ranging from 7 billion to 70 billion parameters and different appliances like LLAMA 2-Chat, which is optimized for dialogue interaction. When it was released in 2023, it outperformed state-of-the-art open source models. A main refinement compared to its predecessor LLAMA, was the focus on transparency and safety including a post-training alignment. \cite{touvron2023llama}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
The \textbf{Llama 2} model was build and released by Meta. It is available in different sizes, ranging from 7 billion to 70 billion parameters and different appliances like LLAMA 2-Chat, which is optimized for dialogue interaction. When it was released in 2023, it outperformed many state-of-the-art open source models. A main refinement compared to its predecessor LLAMA, was the focus on transparency and safety including a post-training alignment. \cite{touvron2023llama}
```

File: paper.tex [hunk 3]
```diff
@@ -89,8 +89,14 @@ \subsection{Generative models}
...
+
+Another generative model for text is \textbf{BLOOM}. This huge 176 billion parameter big model was published in 2023 and is openly available. The model is multilingual and was built in a decentralized manner by many different contributors. Hundreds of researchers helped with the development of the architecture and tokenizer. The final model was trained on the Jean Zay supercomputer. It is built with a decoder-only architecture and used the ROOTS corpus dataset for training. \cite{workshop2023bloom}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
Another generative text model is \textbf{BLOOM}. This huge 176 billion parameter big model was published in 2023 and is openly available. The model is multilingual and was built in a decentralized manner by many different contributors. Hundreds of researchers helped with the development of the architecture and tokenizer. The final model was trained on the Jean Zay supercomputer. It is build on a decoder-only architecture and uses the ROOTS corpus dataset for training. \cite{workshop2023bloom}
```

**Review by EhrenbruderRoschtbraten — COMMENTED** [review]
Some small comments, all in all looks good

**Review by LeviUniAcc — COMMENTED** [review]
i like it, but i think here it is also interesting to know how expensive they are.

**Comment by 1Maxnet1** [comment]
> i like it, but i think here it is also interesting to know how expensive they are.

I think noting whether or not the models are freely available or only behind a paywall, should be sufficient in my opinion. A certain price does imho not make sense, as it can vary depending on different subscription tiers or other conditions and might change over time. So this information might become outdated rather quickly. But I will double check, whether and how I mentioned the availability of the models.

**Comment by LeviUniAcc** [comment]
> > i like it, but i think here it is also interesting to know how expensive they are.
> 
> I think noting whether or not the models are freely available or only behind a paywall, should be sufficient in my opinion. A certain price does imho not make sense, as it can vary depending on different subscription tiers or other conditions and might change over time. So this information might become outdated rather quickly. But I will double check, whether and how I mentioned the availability of the models.

i think we should mention how the current situation is, as this is a big part of the market analysis even if this changes in 2 weeks imo.

**Comment by 1Maxnet1** [comment]
> > > i like it, but i think here it is also interesting to know how expensive they are.
> > 
> > 
> > I think noting whether or not the models are freely available or only behind a paywall, should be sufficient in my opinion. A certain price does imho not make sense, as it can vary depending on different subscription tiers or other conditions and might change over time. So this information might become outdated rather quickly. But I will double check, whether and how I mentioned the availability of the models.
> 
> i think we should mention how the current situation is, as this is a big part of the market analysis even if this changes in 2 weeks imo.

The market analysis was mostly relevant for the competition. Back then I would agree, that this is important, but now all we add is only for the final report and I do not think that they are interested in pricing.

**Review by LeviUniAcc — APPROVED** [review]
lgtm

**Review by EhrenbruderRoschtbraten — APPROVED** [review]
Wird passen denk

**Pull request merged**